source/lovelace/views.markdown

```diff
@@ -52,6 +52,44 @@ views:
       type: string
 {% endconfiguration %}
 
+## Options For Badges
+
+If you define badges as objects instead of strings (by adding `entity:` before entity ID), allowing you to add more customizations:
+
+{% configuration %}
+entity:
+  required: true
+  description: Home Assistant entity ID.
+  type: string
+name:
+  required: false
+  description: Overwrites friendly name.
+  type: string
+icon:
+  required: false
+  description: Overwrites icon or entity picture.
+  type: string
+image:
+  required: false
+  description: The URL of an image.
+  type: string
+{% endconfiguration %}
+
+### Example
+
+View config:
+
+```yaml
+- title: Living room
+  badges:
+    - device_tracker.demo_paulus
```

Comment:
```suggestion
    - entity: device_tracker.demo_paulus
```

Reply:
the `entity` prefix is only required if you wish to set other options for the entity